gopls/doc/subl.md

```diff
@@ -1,81 +1,77 @@
 # Sublime Text
 
-Use the [LSP] package. After installing it using Package Control, do the following:
+Setting up Sublime Text for Golang development.
 
-* Open the **Command Palette**
-* Find and run the command **LSP: Enable Language Server Globally**
-* Select the **gopls** item. Be careful not to select the similarly named *golsp* by mistake.
+## Installation
 
-Finally, you should familiarise yourself with the LSP package's *Settings* and *Key Bindings*. Find them under the menu item **Preferences > Package Settings > LSP**.
+Make sure Go is installed and available in your `PATH`. The [Go Documentation][golang-installation] get Go up and running quickly.
+You can verify that Go is properly installed and available by typing `go help` in a terminal. The command should print a help text instead of an error message.
 
-## Examples
-Minimal global LSP settings, that assume **gopls** and **go** appear on the PATH seen by Sublime Text:<br>
-```
+Use [Package Control] to install the following plugins:
+
+- [LSP] provides language server support in Sublime Text
+- [LSP-gopls] the helper plugin for gopls
```

Comment:
It's better to write "helper package" instead of "helper plugin".

Reply:
Done. Thanks!